packages/osc-ecommerce/app/queries/sanity/fragments/buildUrls.ts

```diff
@@ -19,5 +19,8 @@ export const buildUrls = groq`
     },
     (_type == "product") => {
         "slug": "/${PATHS.PRODUCTS}/" + store.slug.current,
+    },
+    (_type == "award") => {
+        "slug": "/${PATHS.AWARDS}/" + store.slug.current,
```

Comment:
Think you just need to pop the store part off here so it would look like:

```suggestion
        "slug": "/${PATHS.AWARDS}/" + slug.current,
```

The store bit is specific for Shopify pages